css/default.css

```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+input[type="text"] {
+  display: inline-block;
+  margin: 0;
+  width: 40%;
+  font-size: 18px;
+  appearance: none;
+  box-shadow: none;
+  border-radius: none;
+}
+
+input[type="button"] {
+  	border : solid 1px #e0e0e0;
+  	border-radius : 0px 0px 1px 0px ;
+  	moz-border-radius : 0px 0px 1px 0px ;
+  	font-size : 18px;
+  	color : #000000;
+  	padding : 2px 6px;
+    background-color: inherit;
+}
+
+#queryEcho {
+  float: left;
+  padding-top: 20px;
+}
+
+#container ul {
```

Comment:
See if you can give these elements more meaningful names, e.g.
`results-list`
`thumbnail`
`results-item`